schedule/yam/agama_auto_iscsi.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
we try to use `unattended` instead of `auto` everywhere.

Reply:
thanks for review, I updated it.